docs/docs/usage/ide-support.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: doc-page
+title: "IDE support for Dotty"
+---
+
+Dotty comes built-in with the Dotty Language Server, an implementation of the
+[Language Server Protocol](https://github.com/Microsoft/language-server-protocol),
+which means that any editor that implements support for the LSP can get a great
+Dotty integration. Currently, the only IDE we officially support
```

Comment:
"can get a great Dotty integration." => "can work as a Dotty IDE; to the extent of the protocol."